src/coffea/nanoevents/methods/vector.py

```diff
@@ -91,10 +97,39 @@ def delta_r(eta1, phi1, eta2, phi2):
 
 
 behavior = {}
+behavior.update(vector.backends.awkward.behavior)
+
+
+class TwoVectorArray:
```

Comment:
What do these empty classes do?

Reply:
These satisfy the linter and are overwritten by `@awkward.mixin_class(behavior)`, otherwise flake complains the variable doesn't exist since it's patched in and I have to assign the projection classes.

Reply:
changed this to use noqa instead, it's more readable.